### DISCUSSION TITLE
Eero wifi system is blocking Minecraft server

### DISCUSSION THREAD

**Discussion by MrThiago** [itzg/docker-minecraft-bedrock-server#595] [Q&A]
I have tried to add the port forwarding to my Eero system for port 19132. But it still not working.

**Comment by itzg** [comment]
I would recommend investigating with wireshark or tcpdump.

**Comment by MrThiago** [comment]
When the docker is Running the LAN Games does not show anything. However, I managed to connect my phone to the server running on my computer by adding a new game server and adding the computer IP address. 

But If I am in the same LAN, why won't it show the game server under the Friends tab?

**Comment by itzg** [comment]
The LAN discovery mechanism tends to be flaky, I would recommend investigating with wireshark or tcpdump.
